### PR TITLE
Found a few from this gist

### PR DESCRIPTION
**Pull request by whoacowboy**
https://gist.github.com/nocturnalgeek/1b8fa44283314544c487

### CODE REVIEW

**Comment by wesbos** [comment]
thanks a bundle

File: emails.txt [hunk 1]
```diff
@@ -718,23 +997,29 @@ wegwerfmail.org
...
-www.mailinator.com
 wwwnew.eu
 x.ip6.li
 xagloo.com
 xemaps.com
 xents.com
 xmaily.com
 xoxy.net
+xyzfree.net
+yahoo.com.ph
```
Comment (1337) [inline emails.txt]: Were you sure all of these are burner email providers?

Comment (wesbos) [inline emails.txt]: PRobably not yahoo - I'll take them out. LMK if there are others

File: emails.txt [hunk 2]
```diff
@@ -748,8 +1033,9 @@ z1p.biz
 za.com
 zehnminuten.de
 zehnminutenmail.de
+zetmail.com
+zippymail.info
```
Comment (1337) [inline emails.txt]: Are you okay with duplicate list items, like the one right underneath this one?

File: emails.txt (hunk 1 above)
Comment (wesbos) [inline emails.txt]: I see a bunch that should not have been merged  - ill just revert this PR because its garbage

Comment (1337) [inline emails.txt]: Sounds good! Thanks

**Pull request merged**